src/components/CookiesBanner/index.jsx

```diff
@@ -17,6 +17,8 @@ import { mainFontFamily, md, primary, screenSm } from '~/theme/variables'
 import { loadGoogleAnalytics } from '~/utils/googleAnalytics'
 import { loadIntercom } from '~/utils/intercom'
 
+const isDesktop = process.env.REACT_APP_BUILD_FOR_DESKTOP
```

Comment:
This loads as a boolean value? or just a text stating 'true'/'false'?

Reply:
ah, nevermind... if you're only setting it in the scripts, then it shouldn't be an issue. It's set or not.